SU-SWS/stanford_basic
# ARCH-297 GDPR Global Footer Updates

# READY FOR REVIEW

# Summary
- (ensured that links are GDPR approved)
- Updated global footer to match homesite (template and stylesheet updates)
- Removed Decanter breakpoint calls from global footer file (in preparation for breakpoints changing soon)
- Pulled in latest Decanter

# Needed By (Date)
- End of sprint (8/2)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the global footer it working well (can compare closely to stanford.edu)

# Affected Projects or Products
- stanford_basic

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-297

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
